src/components/PortfolioCard.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import styled from 'styled-components';
+
+import { layoutMixins } from '@/styles/layoutMixins';
+
+import { AssetIcon } from './AssetIcon';
+
+type PortfolioCardProps = {
+  assetName: string;
+  assetId: string;
+  actionSlot: React.ReactNode;
+
+  detailLabel: string;
+  detailValue: React.ReactNode;
+};
+
+export const PortfolioCard = ({
```

Comment:
`PortfolioCard` seems too general. Maybe rename it to `VaultPortfolioCard`?

Reply:
Disregard. Maybe even more generic because this PortfolioCard is also used on the TradePage for UnopenedIsolatedPositions